### ISSUE TITLE
export data in stokes widget only exports one profile

### ISSUE THREAD

**Issue by arusstaylor** [CARTAvis/carta-frontend#614]
The export data for the Stokes widget should export all profiles Q, U, P, and PA versus frequency in a single file.  If the display is in fractional polarisation mode then Q, U and P should be in percent.

**Comment by kswang1029** [comment]
@arusstaylor Currently each plot (Q and U, PI, or PA) can be exported individually. Do you want to combine them as separate columns in a singe file? For example ,
<channel>, <velocity>, <frequency>, <Q>, <U>, <PI>, <PA>

**Comment by arusstaylor** [comment]
I can't think of a use case where a user would want only one of the profiles.  Default behaviour should be to export all in a single file with each profile in a separate columns and frequency (the x coordinate) as the first column.  Comma separation is not really required unless we want to import to spreadsheets which is a rare thing for an astronomer to do.  The column headers should be in a row started by #, which his common approach for parsing tables used in astronomy packages.  For the future, another very good option would be to export to a VO table. 

Would be very good also to export the Stokes I profile at the same position into a column. This would  make the exported data more useful for analytics.

**Comment by panchyo0** [comment]
Should we also export all line plots in one file? like Q, U, PI and PA in a png file. if so, should we still keep the ability for user to export image and data as single file?

**Comment by arusstaylor** [comment]
Yes, I think exporting all line plots as a single file is the preferred behaviour.  That is the function of the stokes widget.
